### PR TITLE
Feature/tabs and tabs container component

### PR DESCRIPTION
**Pull request by rakeshkumar-verma**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/_cq_template/.content.xml [hunk 1]
```diff
@@ -6,4 +6,8 @@
     <item_2 jcr:primaryType="nt:unstructured"
             jcr:title="Tab 2"
             sling:resourceType="wcm/foundation/components/responsivegrid"/>
+    <item_3
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/_cq_template/.content.xml:9]: @rakeshkumar-verma : why is this required ? ANy number of layout containers can be added in Tabs

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less [hunk 2]
```diff
@@ -0,0 +1,236 @@
+/* tabs style */
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less:1]: @rakeshkumar-verma @Rupesh-Shende : whys is font color grey on grey bg ? is it as per UX ..

image

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+
+.tabs{
+        
+
+
+    .get_type1_tab(@color,@bgcolor) {
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less]: @Rupesh-Shende @rakeshkumar-verma : the foct icon is white on grey backgroud , please update as its not visible based on active class.

image

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml [hunk 4]
```diff
@@ -32,10 +32,14 @@
                                 cq:policy="hclecomm/components/content/accordion/policy_1588258216814"
                                 jcr:primaryType="nt:unstructured"
                                 sling:resourceType="wcm/core/components/policies/mapping"/>
-							<container
+							              <container
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml]: @rakeshkumar-verma : can we remove the extra space here ?

**Review by rishabh-behl — COMMENTED** [review]
@rakeshkumar-verma @Rupesh-Shende : Pl incorporate comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/_cq_template/.content.xml (hunk 1 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/_cq_template/.content.xml:9]: @rishabh-behl By default 3 tab will be visible, this is as per ACs. Yes content author can add any number of tabs.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less (hunk 2 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less:1]: @rishabh-behl Yes, It is as per UX shared by Ashish.
https://hclo365-my.sharepoint.com/personal/rishabh_b_hcl_com1/_layouts/15/onedrive.aspx?originalPath=aHR0cHM6Ly9oY2xvMzY1LW15LnNoYXJlcG9pbnQuY29tLzpmOi9nL3BlcnNvbmFsL3Jpc2hhYmhfYl9oY2xfY29tMS9FcVdjQUJ4RURkVk9ycU9xUjdTaG9ud0JqM1YzVmROMTcxMUlMX28zN3Npb2N3P3J0aW1lPUIwQXp6UVBoMTBn&id=%2Fpersonal%2Frishabh%5Fb%5Fhcl%5Fcom1%2FDocuments%2FAdobe%2FHCL%2DeCommerce%2FUI%2DUX%2FDesigns%2FTab%2Ftabbing%5F01%2Ejpg&parent=%2Fpersonal%2Frishabh%5Fb%5Fhcl%5Fcom1%2FDocuments%2FAdobe%2FHCL%2DeCommerce%2FUI%2DUX%2FDesigns%2FTab

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml (hunk 4 above)
Comment (rakeshkumar-verma) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml]: @rishabh-behl  Removed extra space.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less (hunk 3 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less]: @rishabh-behl Icon color updated as per UX design.
image

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**